providers/core/core.console.js

```diff
@@ -48,29 +48,23 @@ Logger_console.prototype.print = function (severity, source, msg) {
     return;
   }
   
-  if (typeof process !== 'undefined' &&
-      {}.toString.call(process) === '[object process]' && source) {
-    arr.unshift('\x1B[39m');
```

Comment:
This part is special casing it for the node environment. I think we still want this one.